seedpsql.js

```diff
@@ -102,17 +106,18 @@ const dataGenNeighborhood = (numGen) => {
 
 // definitions
 function writeTenMillionHomeRecords(writer, encoding, callback) {
-    let i = 100;
+    let i = 10000000;
     let id = 0;
+    let neighborhood_id = 1
     function write() {
       let ok = true;
       do {
         i -= 1;
         id += 1;
+        getRandomInt(2) ? neighborhood_id += getRandomInt(2) : neighborhood_id += 0
```

Comment:
To my understanding, this is adding 0 or 1 to neighborhood_id when the first getRandomInt(2) is 1 and adding nothing when the first getRandomInt(2) is 0. For readability, this line could be simplified to `neighborhood_id += getRandom(2) ? getRandom(2) : 0`. But if you would like to, a similar effect of random neighborhood_ids between 1 and 2 could be achieved with ` Math.floor(Math.random() * (max - min + 1)) + min` so `neighborhood_id += Math.floor(Math.random() * 2) + 1 `.